app/controllers/SprintsController.php

```diff
@@ -19,11 +19,17 @@ public function show(Sprint $sprint)
 	public function showWithLiveData(Sprint $sprint)
 	{
 		$phabricator = App::make('phabricator');
-		$currentSprint = $sprint->project->currentSprint();
 		$taskList = new TaskList($phabricator->queryTasksByProject($sprint->phid));
-		$burndown = new BurndownChart($sprint, $taskList, $phabricator->getTaskTransactions($taskList->getClosedTaskIDs()));
 
-		return View::make('sprint.view', compact('sprint', 'currentSprint', 'taskList', 'burndown'));
+		return View::make(
+			'sprint.view',
+			[
+				$sprint,
+				$sprint->project->currentSprint(),
+				$taskList,
+				new BurndownChart($sprint, $taskList, $phabricator->getTaskTransactions($taskList->getClosedTaskIDs()))
+			]
+		);
 	}
```

Comment:
Oh. Does the using code need the key? Or can it also get `[$foo]`? If it needs the key, this change breaks the code.